routers/api/v1/api.go

```diff
@@ -979,10 +988,10 @@ func Routes(sessioner func(http.Handler) http.Handler) *web.Route {
 					m.Get("", repo.GetContentsList)
 					m.Get("/*", repo.GetContents)
 					m.Group("/*", func() {
-						m.Post("", bind(api.CreateFileOptions{}), repo.CreateFile)
```

Comment:
Just a simple todo note: fix the api json objects with a branch in branch list endpoints